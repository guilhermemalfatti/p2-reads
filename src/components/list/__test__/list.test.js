import React from 'react';
import List from '../index';
import renderer from 'react-test-renderer';

test('List render snapshot - comment', () => {
    var item = {
        id: 1,
        voteScore: 5,
        timestamp: 465798
    };

    const component = renderer.create(
        <List
            items={[item]}
            type="comment"
            vote={()=>{}}
            color={()=>{}}
            editComment={()=>{}}
            onDeleteComment={()=>{}}
        />
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

});
