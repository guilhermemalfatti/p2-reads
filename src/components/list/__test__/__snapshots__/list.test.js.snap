// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List render snapshot - comment 1`] = `
<li>
  <div
    className="user-comment undefined"
  >
    <div
      className="comment-votes"
    >
      <div
        className="upvote"
        onClick={[Function]}
      />
      <div
        className="number-of-votes"
      >
        5
      </div>
      <div
        className="downvote"
        onClick={[Function]}
      />
    </div>
    <img
      alt=""
      src="https://cdn1.iconfinder.com/data/icons/flat-business-icons/128/user-32.png"
    />
    <header>
      <a
        className="name"
      />
      <span>
        31/12/69 22:07:45
      </span>
    </header>
    <div
      className="content"
    >
      <p />
    </div>
    <a
      className="edit"
      onClick={[Function]}
    >
      edit
    </a>
    <a
      className="delete"
      onClick={[Function]}
    >
      delete
    </a>
  </div>
</li>
`;
